packages/assets/src/icons/IconPlay.tsx

```diff
@@ -23,27 +23,26 @@ const Svg = (props: React.SVGProps<SVGSVGElement>) => (
 );
 
 type IconProps = {
-  backgroundColor?: ColorProps,
-  circleColor?: ColorProps,
-  color?: ColorProps,
+  backgroundColor?: ColorProps;
+  circleColor?: ColorProps;
+  color?: ColorProps;
 };
-const IconPlay =
-  styled(Svg) <
-  IconProps >
-  (({ backgroundColor, circleColor, color, theme }) => [
-    backgroundColor && {
-      backgroundColor: themed('color', backgroundColor)(theme),
-      padding: backgroundColor ? '0.25em' : 0,
-      overflow: 'visible',
-      borderRadius: '50%',
+const IconPlay = styled(Svg).withConfig({
```

Comment:
these icons are generated automatically using SVGR, you have to change the template to actually change the file

Reply:
check this https://github.com/coingaming/moon-design/blob/develop/packages/assets/.config/templates/svgr.template.icons.js